s.py

```diff
@@ -0,0 +1,229 @@
+import csv
+from dataclasses import dataclass
+from typing import Optional, List, Union, Dict
+
+import requests
+from bs4 import BeautifulSoup
+
+
```

Comment:
It's about time you've learnt how to do the actual logging
```suggestion
import logging
logging.basicConfig(level=logging.INFO)
log = logging.getLogger(__name__)
```